ignite/config/testdata/testdata.go

```diff
@@ -17,10 +20,16 @@ var Versions = map[chainconfig.Version][]byte{
 
 var NetworkConfig = networkconfigTestdata.ConfigYAML
 
+var PluginsConfig = pluginsconfigTestdata.ConfigYAML
+
 func GetLatestConfig(t *testing.T) *config.ChainConfig {
 	return v1testdata.GetConfig(t)
 }
 
 func GetLatestNetworkConfig(t *testing.T) *config.ChainConfig {
 	return networkconfigTestdata.GetConfig(t)
 }
+
+func GetPluginsConfig(t *testing.T) *pluginsconfig.Config {
+	return pluginsconfigTestdata.GetConfig(t)
+}
```

Comment:
Do we really need to declare the plugins testdata here ?

The current usage is 
`config/plugins/parse_test.go` calls `config/testdata/testdata.go` which calls `config/plugins/testdata/testdata.go`

We can remove the man in the middle and call directly `config/plugins/testdata/testdata.go` from `config/plugins/parse_test.go` right ?